dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -331,8 +331,8 @@ def test_profiled_histogram(self):
         # with equal bin size, each bin has the width of 1
         df3 = pd.Series(["1.0", "1.0", "3.0", "4.0"])
         expected_histogram3 = {
-            'bin_counts': np.array([2, 0, 2]),
-            'bin_edges': np.array([1.0, 2.0, 3.0, 4.0]),
+            'bin_counts': np.array([2, 0, 1, 1]),
+            'bin_edges': np.array([1.0, 1.75, 2.5, 3.25, 4.0]),
```

Comment:
shouldn't we be comparing to numpy histograms?

Reply:
which histogram method? all might choose a different result. Depends on the loss function we want to achieve.

Reply:
You can check via almost equal https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual

Reply:
Almost equal wouldn't work here as the histogram itself is different